### PR TITLE
Hw5

### PR DESCRIPTION
**Pull request by bwilder95**
Docker Assignment

### CODE REVIEW

File: scripts/baseball-logic.sh [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+# copy baseball.sql database to container
+docker cp baseball.sql mysql-container:/baseball.sql
+
+# select baseball_db
+echo "**********USE baseball_db**********"
+docker exec mysql-container bash -c "mysql --user=root --password=password123 --database=baseball_db -e 'USE baseball_db'" # pragma: allowlist secret
+
+# upload data (this step take a while)
+echo "**********SOURCE baseball.sql**********..... this step takes about 15 minutes*******"
+docker exec mysql-container bash -c "mysql --user=root --password=password123 --database=baseball_db -e 'SOURCE baseball.sql'" # pragma: allowlist secret
```
Comment (gramadin) [inline scripts/baseball-logic.sh:12]: i think copying baseball.sql in and then using `mysql baseball_db < baseball.sql -ppasword123` may be faster.

File: scripts/baseball-logic.sh [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+CREATE TABLE batting_avg_annual \
+AS (SELECT batter_counts.batter,batter_counts.Hit,batter_counts.atBat,game.local_date  \
+FROM batter_counts \
+INNER JOIN game ON batter_counts.game_id=game.game_id);"
+
+# Creating annual batter average table
+echo "**********output new table to storage**********"
+docker exec mysql-container /docker-entrypoint-initdb.d/dump.sql -u root --password=password123 baseball_db > dump.sql
+
+# Showing first 20 rows of Batter average
```
Comment (gramadin) [inline scripts/baseball-logic.sh:39]: I think the results should be only the players in game 12560.
try a nested query to only show the batters who are in game 12560 from the rolling 100 table that you would end up making. 
> *use something like this in your where statement:
>      `select batter from batter_counts where game_id = '12560'`

File: dump.sql [hunk 3]
```diff
@@ -0,0 +1 @@
+OCI runtime exec failed: exec failed: container_linux.go:349: starting container process caused "exec: \"/docker-entrypoint-initdb.d/dump.sql\": permission denied": unknown
```
[the remark this answers is not in the document]
Comment (bwilder95) [reply dump.sql:1]: I am confused by what you mean, but yes my password is that. password123. I'm not sure why it is not letting me store my volume.

File: requirements.dev.txt [hunk 4]
```diff
@@ -7,12 +7,16 @@
...
 argon2-cffi==20.1.0       # via notebook
 async-generator==1.10     # via nbclient
-attrs==20.2.0             # via jsonschema
+attrs==20.3.0             # via jsonschema
 backcall==0.2.0           # via ipython
 black==20.8b1             # via -r requirements.dev.in
 bleach==3.2.1             # via nbconvert
-certifi==2020.6.20        # via requests
+certifi==2020.11.8        # via requests
+<<<<<<< HEAD
```
Comment (dafrenchyman) [inline requirements.dev.txt:15]: Looks like you accidentally committed some merge conflicts here. You should probably just regenerate the file to fix this.

**Pull request merged**